feature/gribi/otg_tests/gribi_flush/README.md

```diff
@@ -0,0 +1,165 @@
+TE-18.4 - gRIBI Flush operation with Live traffic
+
+## Summary
+This test verifies when gRIBI flush operation is issued all the gRIBI rules are removed leaving the static routes intact.
+Ongoing traffic flows using gRIBI rules will switch to static route.
+
+## Topology
+
+* [`featureprofiles/topologies/atedut_2.testbed`](https://github.com/openconfig/featureprofiles/blob/main/topologies/atedut_2.testbed)
+
+## Test setup
+
+TODO: Complete test environment setup steps
+
+inner_ipv6_dst_A = "2001:aa:bb::1/128"
+inner_ipv6_dst_B = "2001:aa:bb::2/128"
+inner_ipv6_default = "::/0"
+
+ipv4_inner_dst_A = "10.5.1.1/32"
+ipv4_inner_dst_B = "10.5.1.2/32"
+ipv4_inner_default = "0.0.0.0/0"
+
+outer_ipv6_src =      "2001:f:a:1::0"
+outer_ipv6_dst_A =    "2001:f:c:e::1"
+outer_ipv6_dst_B =    "2001:f:c:e::2"
+outer_ipv6_dst_def =  "2001:1:1:1::0"
+outer_dst_udp_port =  "6635"
+outer_dscp =          "26"
+outer_ip-ttl =        "64"
+
+## Procedure
+
+### TE-18.4.1 Perform gRIBI flush operation to verify traffic switches to backup prefix match rule.
+
+#### gRIBI RPC content
+
+The gRIBI client should send this proto message to the DUT to create AFT
+entries.
+
+```proto
+#
+# aft entries used for network instance "NI_A"
+IPv6Entry {2001:DB8:2::2/128 (NI_A)} -> NHG#100 (DEFAULT VRF)
+IPv4Entry {203.0.113.2/32 (NI_A)} -> NHG#100 (DEFAULT VRF) -> {
+  {NH#101, DEFAULT VRF}
+}
+
+# this nexthop specifies a MPLS in UDP encapsulation
+NH#101 -> {
+  encap_-_headers {
+    encap_header {
+      index: 1
+      mpls {
+        pushed_mpls_label_stack: [101,]
+      }
+    }
+    encap_header {
+      index: 2
+      udp_v6 {
+        src_ip: "outer_ipv6_src"
+        dst_ip: "outer_ipv6_dst_A"
+        dst_udp_port: "outer_dst_udp_port"
+        ip_ttl: "outer_ip-ttl"
+        dscp: "outer_dscp"
+      }
+    }
+  }
+  next_hop_group_id: "nhg_A"  # new OC path /network-instances/network-instance/afts/next-hop-groups/next-hop-group/state/
+  network_instance: "DEFAULT"
+}
+
+#
+# entries used for network-instance "NI_B"
+IPv6Entry {2001:DB8:2::2/128 (NI_B)} -> NHG#200 (DEFAULT VRF)
+IPv4Entry {203.0.113.2/32 (NI_B)} -> NHG#200 (DEFAULT VRF) -> {
+  {NH#201, DEFAULT VRF}
+}
+
+NH#201 -> {
+  encap_headers {
+    encap_header {
+      index: 1
+      mpls {
+        pushed_mpls_label_stack: [201,]
+      }
+    }
+    encap_header {
+      index: 2
+      udp_v6 {
+        src_ip: "outer_ipv6_src"
+        dst_ip: "outer_ipv6_dst_B"
+        dst_udp_port: "outer_dst_udp_port"
+        ip_ttl: "outer_ip-ttl"
+        dscp: "outer_dscp"
+      }
+    }
+  }
+  next_hop_group_id: "nhg_B"  
+  # network_instance: "DEFAULT"  TODO: requires new OC path /network-instances/network-instance/afts/next-hop-groups/next-hop-group/state/network-instance
+}
+```
+
+* Configure static-routes with next-hops using encap-headers as a backup prefix match rule for MPLS in GRE encap.
```

Comment:
The canonical OC in JSON format should be provided.  I think you can reference [PF-1.14](https://github.com/openconfig/featureprofiles/tree/dd158fd775fe977ebb9d79ca1ac720846b1d41e1/feature/policy_forwarding/otg_tests/mpls_gre_ipv4_encap_test ) for this.  It uses policy-forwarding for the match rules with encap-headers though, not static routes.